### ISSUE TITLE
Fraction exhaled nitric oxide elevated

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10894]
**Preferred term label:**
Fraction exhaled nitric oxide elevated

**Synonyms**
FeNO

**Definition (free text, please give PubMed ID)**
https://pmc.ncbi.nlm.nih.gov/articles/PMC6690504/

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0002795

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Asthma
Eosinophilic pneumonia
COPD
Pulmonary fibrosis
Interstitial lung disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001236
- label: Elevated fraction exhaled nitric oxide
- parents: [Abnormal respiratory system physiology (HP:0002795)](https://hpo.jax.org/app/browse/term/HP:0002795)
- pmid: PMID:31417809
- def: The fractional exhaled nitric oxide (FeNO) is elevated above the upper limit of normal. FeNO is an endogenous gaseous molecule which can be measured in the human breath test because of airway inflammation. It has been studied extensively as a marker of inflammation and has been incorporated into an algorithm for asthma management.
- comment: Fractional exhaled nitric oxide (FeNO) is a simple, reproducible, and a noninvasive marker of inflammation that may be elevated in diseases such as ssthma, eosinophilic pneumonia, COPD, pulmonary fibrosis, and interstitial lung disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10894
- synonyms: Fraction exhaled nitric oxide elevated(NONE)

**Comment by MickeySegal** [comment]
Typo: "diseases such as ssthma"
